.changeset/dirty-eels-poke.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/backend-common': patch
```

Comment:
Just wondering if this should be tagged as a minor change (as backend-common version is still < 1.0.0), because in the really rare scenario that someone has explicitly removed the cache config, there is a likelihood of a change in behavior with this change.

Reply:
Maybe I'm also reading the code wrong, but it looks like you've also removed the NoStore option. Do we need to document that NoStore is no longer possible/provided?

Reply:
It is a breaking change to fix broken behavior? 🤔 Is someone out there relying on caching *not* working?
https://xkcd.com/1172/

To the second comment, I would feel the need to document it if one could have configured NoStore affirmatively; but that [isn't an option](https://github.com/backstage/backstage/blob/3c0fa36d4d7c8f72bbf23d94098ca94302b68748/packages/backend-common/config.d.ts#L137). This implies that someone _removed the default configuration_ of in-memory, then is relying on a lack of configuration to have a no-op cache. This seems like an unreasonable position to me, but again — relevant XKCD. I'm curious to get maintainer feedback on this!